### PR TITLE
feat: detail tab

### PR DESCRIPTION
**Pull request by bilalcorbacioglu**
## Summary

- Detail Tab integration
- Earn widget (Show fees)
- It doesn't have `inverted` features which Uniswap has. We don't want to use this feature for now. But I added it anyway. Probably we will turn it on soon.

### CODE REVIEW

File: src/hooks/concentratedLiquidity/hooks/evm.ts [hunk 1]
```diff
@@ -310,3 +300,125 @@ export function usePoolTVL(token0: Token | undefined, token1: Token | undefined)
...
+  const { result: token1result } = useSingleCallResult(token1Contract ? token1Contract : undefined, 'balanceOf', [
+    poolAddress,
+  ]);
+
+  if (!token0 || !token1 || !fee) return [undefined, undefined];
+  if (!poolAddress) return [undefined, undefined];
+
+  const underlyingToken0 = token0result ? new TokenAmount(token0, token0result[0]) : undefined;
+  const underlyingToken1 = token1result ? new TokenAmount(token1, token1result[0]) : undefined;
+  return [underlyingToken0, underlyingToken1];
```
Comment (SarjuHansaliya) [inline src/hooks/concentratedLiquidity/hooks/evm.ts:329]: need to use useMemo here
```ts
return useMemo(() => {
  const underlyingToken0 = token0result ? new TokenAmount(token0, token0result[0]) : undefined;
  const underlyingToken1 = token1result ? new TokenAmount(token1, token1result[0]) : undefined;
  return [underlyingToken0, underlyingToken1];
}, [token0, token1, token0Result, token1result])
```

File: src/hooks/concentratedLiquidity/hooks/evm.ts [hunk 2]
```diff
@@ -310,3 +300,125 @@ export function usePoolTVL(token0: Token | undefined, token1: Token | undefined)
...
+  }, [positionManager, tokenIdHexString, owner, latestBlockNumber]);
+
+  if (pool && amounts) {
+    return [
+      CurrencyAmount.fromRawAmount(unwrappedToken(pool.token0, chainId), amounts[0].toString()),
+      CurrencyAmount.fromRawAmount(unwrappedToken(pool.token1, chainId), amounts[1].toString()),
+    ];
+  } else {
+    return [undefined, undefined];
+  }
```
Comment (SarjuHansaliya) [inline src/hooks/concentratedLiquidity/hooks/evm.ts:373]: need to use `useMemo` as I mentioned above

File: src/components/ConcentratedLiquidity/DetailModal/Tabs/DetailTab/index.tsx [hunk 3]
```diff
@@ -1,79 +1,181 @@
...
+  // TODO: Open it in production
+  // -------------------------------------------------------
+  // const useUSDCPrice = useUSDCPriceHook[chainId];
+  // const price0 = useUSDCPrice(position?.token0 ?? undefined);
+  // const price1 = useUSDCPrice(position?.token1 ?? undefined);
+  // -------------------------------------------------------
+  const USDC = USDCe[chainId];
+  const price0 = position?.token0 && new Price(position.token0, USDC, '1', '1');
+  const price1 = position?.token1 && new Price(position.token1, USDC, '1', '1');
+  // -------------------------------------------------------
```
Comment (SarjuHansaliya) [inline src/components/ConcentratedLiquidity/DetailModal/Tabs/DetailTab/index.tsx:78]: should we fix this now ? is `useUSDCPrice` hook creating issue in fuji ?

**Review by SarjuHansaliya — CHANGES_REQUESTED** [review]

File: src/components/ConcentratedLiquidity/DetailModal/Tabs/DetailTab/index.tsx (hunk 3 above)
Comment (bilalcorbacioglu) [reply src/components/ConcentratedLiquidity/DetailModal/Tabs/DetailTab/index.tsx:78]: I've opened it.

Comment (SarjuHansaliya) [reply src/components/ConcentratedLiquidity/DetailModal/Tabs/DetailTab/index.tsx]: yes thats fine but atleast we have code in place so we can use test it in mainnet

**Pull request merged**